lib/commonmark.js

```diff
@@ -67,7 +66,9 @@ function renderHtml(markdownString, options, cb) {
   }
 
   if (options.sanitize) {
-    result = sanitizer.sanitize(result, uriPolicy);
+    dompurify.addHook('afterSanitizeAttributes', sanitize);
```

Comment:
Please move all the `dompurify` use to the `sanitize.js`. Perhaps even nicer would be to expect the `options.sanitize` to be a functor which could be passed from outside. (would export the sanitize function from the `lib/index.js` to handle it. That way the code could be tree-shaken away when not used by the app.

```suggestion
if (typeof options.sanitize === 'function') {
   result = options.sanitize(result);
}
```